src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -216,8 +216,10 @@ public static void assertRegularCommandSuccess(Command command, Model actualMode
      * that takes a string {@code expectedMessage} and a Person {@code expectedPerson}.
      */
     public static void assertDisplayableCommandSuccess(Command command, Model actualModel, String expectedMessage,
-                                                       Model expectedModel, Person expectedPerson) {
-        DisplayableCommandResult expectedCommandResult = new DisplayableCommandResult(expectedMessage, expectedPerson);
+                                                       Model expectedModel, Person expectedPerson,
+                                                       String expectedDisplayMessage) {
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(